htdocs/install/mysql/data/llx_c_type_contact.sql

```diff
@@ -118,3 +118,6 @@ insert into llx_c_type_contact (element, source, code, libelle, active ) values
 insert into llx_c_type_contact (element, source, code, libelle, active ) values ('conferenceorbooth', 'internal', 'MANAGER',      'Conference or Booth manager', 1);
 insert into llx_c_type_contact (element, source, code, libelle, active ) values ('conferenceorbooth', 'external', 'SPEAKER',      'Conference Speaker', 1);
 insert into llx_c_type_contact (element, source, code, libelle, active ) values ('conferenceorbooth', 'external', 'RESPONSIBLE',  'Booth responsible', 1);
+
+-- Thirdparty
+insert into llx_c_type_contact (element, source, code, libelle, active ) values ('societe', 'internal', 'SALESREPTHIRD',  'Sales Representative', 1);
```

Comment:
Managing Sales Representative using contact is inconflict with manage Sale Representative using the table llx_societe_commerciaux.
What is goal of PR: Managing using llx_societe_commerciaux with extended information or managing using llx_c_type_contact ?